src/lang/en-US.json

```diff
@@ -3,6 +3,12 @@
   "Countdown-fourth-list-item": "SECONDS",
   "Countdown-second-list-item": "HOURS",
   "Countdown-third-list-item": "MINUTES",
+  "FAQ-list-answer1": "Lorem ipsum dolor sit amet, consectetur adipiscing elit  Suspendisse malesuada lacus ex, sit amet blandit leo lobortis eget.",
```

Comment:
:wrench: We don't need six identical translations.

Reply:
Addressed: commit ee174247904fe884c47cfb36e579dd8e0ed3d65e